django/contrib/admin/options.py

```diff
@@ -829,6 +829,15 @@ def get_action(self, action):
             description = capfirst(action.replace('_', ' '))
         return func, action, description
 
+    def get_empty_value_display(self):
```

Comment:
The thing I don't like about defining an attribute on the field, it's way more work to override an entire field to just change the label for an empty value. And furthermore you'd need multiple form field classes for different empty values.

On March 13, 2015 1:17:30 PM GMT+01:00, Tim Graham <notifications@github.com> wrote:
>> @@ -829,6 +829,15 @@ def get_action(self, action):
>>              description = capfirst(action.replace('_', ' '))
>>          return func, action, description
>>  
>> +    def get_empty_value_display(self):
>
>+1 to field property, I thought of that also.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/django/django/pull/4307/files#r26381000